src/cmd/wg_key.rs

```diff
@@ -0,0 +1,14 @@
+use crate::cmd::{Cmd};
+use crate::types::{WgPubkey};
+use serde::{Deserialize, Serialize};
+
+#[derive(Debug, Serialize, Deserialize, Clone)]
+pub struct RegisterWgKey {
+    public_key: WgPubkey,
+}
+
+impl Cmd for RegisterWgKey {
+    type Output = ();
+    const METHOD: http::Method = http::Method::POST;
+    const PATH: &'static str = "wg-key";
```

Comment:
```suggestion
    const PATH: &'static str = "wg_key";
```

For consistency with other paths.

Although I would prefer to be explicit about caching for now, maybe `cache_wg_key`? But that isn't a big deal.

Reply:
done